command/src/channel.rs

```diff
@@ -537,17 +528,17 @@ mod tests {
 
         let message_to_send = Serializable(42);
 
-        nonblocking_channel.blocking();
-        nonblocking_channel.write_message(&message_to_send);
+        nonblocking_channel.blocking().unwrap();
+        nonblocking_channel.write_message(&message_to_send).unwrap();
 
         println!("we wrote a message!");
 
         println!("reading message..");
         // blocking_channel.readable();
-        let message = blocking_channel.read_message();
+        let message = blocking_channel.read_message().unwrap();
```

Comment:
To me, this should be replaced by an `expect`

Reply:
done